numpy/distutils/misc_util.py

```diff
@@ -2362,6 +2362,29 @@ def show():
                 * ``not found``: dispatched features that are not supported
                   in the system
 
+                NumPy BLAS/LAPACK Installation Notes
+                ------------------------------------
+                Installing a numpy wheel (e.g. ``pip install numpy``) includes
+                an OpenBLAS implementation of the BLAS and LAPACK linear algebra
+                APIs. In this case, ``library_dirs`` reports the build time
+                configuration; the OpenBLAS library is actually in
+                ``site-packages/numpy.libs/`` or
+                ``site-packages/numpy/.dylibs/`` and it's compiled with
+                gcc/gfortran.
+
+                Installing numpy from source (e.g.
+                ``pip install numpy --no-binary numpy``) looks for BLAS and
+                LAPACK dynamic link libraries at build time. This search is
+                influenced by the environment variables
+                NPY_BLAS_ORDER/NPY_LAPACK_ORDER, if set, and the file
+                ``~/.numpy-site.cfg``, if present.
+                On macOS, 'accelerate' (Apple's Accelerate BLAS library) is in
+                the default search order (but not in numpy 1.20) after
+                'openblas'.
+                Loading a numpy library that's linked to Accelerate on
+                macOS < 11.3 will detect an Accelerate bug and raise a
+                RuntimeError.
```

Comment:
Maybe leave this part out? People who hit this problem will not be looking for documentation of it in this docstring, they will hit [this error message](https://github.com/numpy/numpy/blob/v1.21.0/numpy/__init__.py#L385) first
```suggestion
```